src/woo_publications/metadata/admin.py

```diff
@@ -10,8 +10,11 @@
 
 @admin.register(InformationCategory)
 class InformationCategoryAdmin(OrderedModelAdmin):
-    list_display = ("naam", "identifier", "oorsprong", "move_up_down_links")
-    readonly_fields = ("oorsprong",)
+    list_display = ("uuid", "naam", "identifier", "oorsprong", "move_up_down_links")
```

Comment:
I don't think we need to include the UUID in the list_display tbh, if you need to know it -> go to the detail view/change form

it's a long string that creates clutter with little added value